libs/bn-common/src/components/BoatnetCommonInputComponent.vue

```diff
@@ -77,12 +80,24 @@ export default createComponent({
 
     const valueHolder = computed({
       get: () => {
-        return get(props.model, props.config ? props.config.modelName : '');
+        const modelName = props.config ? props.config.modelName : '';
+        if (typeof modelName === 'object') {
+          return get(props.model, modelName[0]);
+        } else {
+          return get(props.model, modelName);
+        }
       },
       set: (val: any) => {
         const modelName = props.config ? props.config.modelName : '';
-        const fields = modelName.split('.');
-        setValue(props.model, fields, val);
+        if (typeof modelName === 'object') {
```

Comment:
In this example when setting depth units, it needs to be applied to two fields, fishing depth and bottom depth. I created an array in the config where you can specify multiple fields to set the value of. But it makes the code a bit messy since I have to add these if statements. Not sure if I should create a separate component instead or open to any suggestions  
![image](https://user-images.githubusercontent.com/44685558/71422444-db721e00-2636-11ea-8ce5-3695ae22b43b.png)

Reply:
I could see a root level section of the config file that sets app wide defaults.  Something like:

"defaults": [
  "bottomDepth": "fathoms",
  "fishingDepth": "meters",
  "flowScaleCatch.measurement.type": "weight",
  "flowScaleCatch.measurement.units": "kg",
  ...
]

Then selects and toggles can check the defaults array to see if there's a default that should be applied.